cql/DashboardLibrary.cql

```diff
@@ -182,78 +195,71 @@ define function CytologyInterpretation(conclusions List<System.Concept>):
         when c ~ "Endocervical Cells Favor Neoplasia" then {text: 'Endocervical Cells Favor Neoplasia', rank: 1}        
         when c ~ "Atypical Endometrial Cells" then {text: 'Atypical Endometrial Cells', rank: 1}
 
-        when c ~ "Histiocytes" then {text: 'Histiocytes', rank: 1}
-        when c ~ "Benign Endometrial Cells" then {text: 'Benign Endometrial Cells', rank: 1}
-        when c ~ "Benign Glandular Cells" then {text: 'Benign Glandular Cells', rank: 1}
-        when c ~ "Endometrial Stromal Cells" then {text: 'Endometrial stromal cells', rank: 1}
-        when c ~ "Unsatisfactory" then {text: 'Unsatisfactory', rank: 1}
-        when c ~ "Absent Transformation Zone" then {text: 'Absent Transformation Zone', rank: 1}
+        when c ~ "Histiocytes" then {riskTableInput: 'HIST', label: 'Histiocytes', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Benign Endometrial Cells" then {riskTableInput: 'BEMC', label: 'Benign Endometrial Cells', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Benign Glandular Cells" then {riskTableInput: 'BGC', label: 'Benign Glandular Cells', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Endometrial Stromal Cells" then {riskTableInput: 'EMSC', label: 'Endometrial stromal cells', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Unsatisfactory" then {riskTableInput: 'UNSAT', label: 'Unsatisfactory', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Absent Transformation Zone" then {riskTableInput: 'ATZ', label: 'Absent Transformation Zone', rank: 1, grade: highlightGrades.normal}
+        when c ~ "Insufficient" then {riskTableInput: 'Insufficient', label: 'Insufficient/Unsatisfactory', rank: 1, grade: highlightGrades.unrecognized}
 
-        when c is null then {text: 'NULL', rank: -1}
-        else {text: 'UNK', rank: 1}
+        when c is null then {riskTableInput: 'NULL', label: 'No result found', rank: -1, grade: highlightGrades.empty}
+        else {riskTableInput: 'UNK', label: 'Unrecognized Test Result', rank: 1, grade: highlightGrades.unrecognized}
```

Comment:
I may be getting confused by the commits and diff, but I thought we were only adding an "Insufficient" interpretation to `BiopsyInterpretation`, not `CytologyInterpretation`. Has it been indicated that cytology tests can also have a result of "Insufficient?

Reply:
My mistake. The task description said "Any **biopsy** with a result SNOMED value of 112631006". I will fix this.

Reply:
All good! I must have missed that in my initial review. I believe we should add "Insufficient" to just the BiopsyInterpretation, not CytologyInterpretation since:
1. In the Data Mapping spreadsheet we looked at yesterday, the `112631006` "Insufficient" SNOMED-CT code only shows up as the result of a histology/biopsy test, not a cytology test.
2. Cytology already has an "Unsatisfactory" interpretation, which serves a similar purpose and actually has a guidelines recommendation for it.